plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraConnectorTest.java

```diff
@@ -909,6 +909,43 @@ public void testClusteringKeyPushdownInequality()
         }
     }
 
+    @Test
+    void testMultiColumnKey()
+    {
+        String tableName = "test_multi_column_key" + randomNameSuffix();
+        onCassandra("CREATE TABLE tpch." + tableName + "(user_id text, key text, updated_at timestamp, value text, PRIMARY KEY (user_id, key, updated_at))");
```

Comment:
nit: How about using test table here ?

Reply:
I'm not a big fun of `TestCassandraTable`  to be honest. The readability is less than hard-coded query and we have to imagine the generated CQL.